### ISSUE TITLE
'stty' is not recognized as an internal or external command, operable program or batch file.

### ISSUE THREAD

**Issue by swiftyy-mage** [hbmartin/pytube3#13]
Windows specific problem. In cli.py, the way the terminal size is determined uses `stty`, which is only for unix-derived systems. Using the CLI on Windows 10 (the only one I can test) with a fresh venv having just downloaded pytube3 (but also having tested seperately using the PR #11) as such:

    C:\Users\anon\Desktop>pytube3 https://www.youtube.com/watch?v=rVHR0UPHERQ --itag 18

returns this:

    A Superb Case of Villainy.mp4 | 11631467 bytes
    'stty' is not recognized as an internal or external command,
    operable program or batch file.
    Traceback (most recent call last):
      File "C:\Users\anon\AppData\Local\Programs\Python\Python36\lib\runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
      File "C:\Users\anon\AppData\Local\Programs\Python\Python36\lib\runpy.py", line 85, in _run_code
    exec(code, run_globals)
      File "C:\Users\anon\Desktop\tv\Scripts\pytube3.exe\__main__.py", line 9, in <module>
      File "c:\users\anon\desktop\tv\lib\site-packages\pytube\cli.py", line 39, in main
    download(youtube=youtube, itag=args.itag)
      File "c:\users\anon\desktop\tv\lib\site-packages\pytube\cli.py", line 189, in download
    stream.download()
      File "c:\users\anon\desktop\tv\lib\site-packages\pytube\streams.py", line 250, in download
    self.on_progress(chunk, fh, bytes_remaining)
      File "c:\users\anon\desktop\tv\lib\site-packages\pytube\streams.py", line 304, in on_progress
    on_progress(self, chunk, file_handler, bytes_remaining)
      File "c:\users\anon\desktop\tv\lib\site-packages\pytube\cli.py", line 165, in on_progress
    display_progress_bar(bytes_received, filesize)
      File "c:\users\anon\desktop\tv\lib\site-packages\pytube\cli.py", line 146, in display_progress_bar
    _, columns = get_terminal_size()
      File "c:\users\anon\desktop\tv\lib\site-packages\pytube\cli.py", line 121, in get_terminal_size
    rows, columns = os.popen("stty size", "r").read().split()
    ValueError: not enough values to unpack (expected 2, got 0)

Will submit a PR in a few minutes which works on Windows and Linux.

**Comment by swiftyy-mage** [comment]
Okay so this repo is forked from my repo, and honestly I never bothered to properly learn how github works so submitting a pull request might take more than a few minutes. In the meantime here is the fix. Change the `get_terminal_size` function in cli.py from

    def get_terminal_size() -> Tuple[int, int]:
        """Return the terminal size in rows and columns."""
        rows, columns = os.popen("stty size", "r").read().split()
        return int(rows), int(columns)

to 

    def get_terminal_size() -> Tuple[int, int]:
        """Return the terminal size in rows and columns."""
        dims = os.get_terminal_size()
        rows, columns = dims.lines, dims.columns
        return rows, columns
. Obviously we wouldn't even need to define the function `get_terminal_size` because the module `os` provides a function of exactly the same name, so feel free to get rid of it and just call `os.get_terminal_size().columns.` from within `display_progress_bar`. Tested on Windows 10 and Linux Mint 19.2 and it works fine. Someone else would have to test MacOS but I don't forsee any issues. Another issue I noticed is that the ↳ symbol in `display_progress_bar` is not properly shown in the Windows CLI. I'll figure out how to submit a PR the correct way and then see if I can get the ↳ symbol working.
